.circleci/config.yml

```diff
@@ -774,6 +774,24 @@ jobs:
       - run: "${KURTOSIS_BINPATH} engine restart"
       - run: "${KURTOSIS_BINPATH} engine status"
 
+      # Test APIC restart
+      - when:
+          condition:
+            equal: [ "docker", << parameters.cli-cluster-backend >> ]
+          steps:
+            - run:
+                name: Test APIC restart
+                command: |
+                  ${KURTOSIS_BINPATH} enclave add -n test-apic-restart
+                  ${KURTOSIS_BINPATH} enclave inspect --full-uuids test-apic-restart > /tmp/inspect_output.txt
+                  inspect_output_uuid_line=$(grep "^UUID:" /tmp/inspect_output.txt) 
+                  enclave_uuid=$(echo ${inspect_output_uuid_line:17:54})
+                  ${KURTOSIS_BINPATH} service add test-apic-restart service-test kurtosistech/example-datastore-server
+                  apic_container_id=$(docker ps --filter "label=com.kurtosistech.container-type=api-container" --filter "label=com.kurtosistech.enclave-id=$(echo $enclave_uuid)" --format "{{.ID}}")
```

Comment:
perhaps we can add an APIC restart command in the future!

Reply:
@leoporoli contemplated it, but I thought this was another implementation detail exposed. Ideally APIC (and in the future, engine) is transparent.